corehq/pillows/mappings/user_mapping.py

```diff
@@ -3,7 +3,7 @@
 
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-USER_INDEX = es_index("hqusers_2017-03-22")
+USER_INDEX = es_index("hqusers_2017-04-24")
```

Comment:
if you are not changing the mapping file you should not need to change the index name. i think you can just run the management command `./manage.py ptop_reindexer_v2 user --in-place` which will reindex all the docs without  creating an entirely new index. The other will create a whole new index (and hold up deploy i believe). Let me know if thats not actually what you were going for. If you do proceed with changing the name, I think there are a few other places it appears in the code base (particularly test code) that you will have to change.